find_missing.py

```
@@ -0,0 +1,24 @@
+import xml.etree.ElementTree as ET
+import os
+
+def get_strings(path):
+    try:
+        tree = ET.parse(path)
+        root = tree.getroot()
+        return {child.attrib['name']: child.text for child in root if child.tag == 'string'}
+    except:
```

Ruff (0.15.12)

[error] Do not use bare `except`

(E722)